services/speech_to_text/v1.js

```diff
@@ -42,17 +42,17 @@ module.exports = function (RED) {
 
     this.on('input', function (msg) {
       if (!msg.payload) {
-        var message = 'Missing property: msg.payload';
-        node.error(message, msg)
+        var message_err_payload = 'Missing property: msg.payload';
+        node.error(message_err_payload, msg)
         return;
       }
 
       username = username || this.credentials.username;
       password = password || this.credentials.password;
 
       if (!username || !password) {
-        var message = 'Missing Speech To Text service credentials';
-        node.error(message, msg)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'message_err_credentials' is not in camel case.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2391504090/issues/source?bid=3225115&fileBranchId=3229003#l54)